### DISCUSSION TITLE
Cockpit with Letsencrypt documentation incomplete

### DISCUSSION THREAD

**Discussion by m4l490n** [cockpit-project/cockpit#16988] [Q&A]
### Explain what happens

The documentation found in [Cockpit with LetsEncrypt ](https://github.com/cockpit-project/cockpit/wiki/Cockpit-with-LetsEncrypt) is incomplete. It only tells to create a bash script and give it execution permissions. But then what? Do I run the script? or what am I supposed to do with that information?

### Version of Cockpit

188

### Where is the problem in Cockpit?

Documentation

### Server operating system

Debian

### Server operating system version

Debian 10

### What browsers are you using?

Chrome

**Comment by jelly** [comment]
I believe the intention of the script is to run after a certbot renewal so that the certificates are in the correct place for cockpit to pick up.

**Reply by martinpitt** [reply]
Correct -- certbot itself runs that script whenever it refreshes its certificates, it's a "hook" directory.

**Reply by m4l490n** [reply]
Could I verify this with a renewal dry run?

**Reply by m4l490n** [reply]
I just did a ` sudo certbot renew --dry-run` and nothing happened. What am I missing?

**Reply by martinpitt** [reply]
Supposedly a dry run wouldn't actually run the hooks, as it then wouldn't be very "dry" any more?

**Reply by martinpitt** [reply]
Any updates on this?
